Framework/Crystal/src/FindUBUsingLatticeParameters.cpp

```diff
@@ -91,18 +95,6 @@ void FindUBUsingLatticeParameters::exec() {
     g_log.notice(std::string("UB NOT SAVED."));
   } else // tell user how many would be indexed
   {      // and save the UB in the sample
-
-    std::vector<double> sigabc(7);
-    std::vector<V3D> miller_ind;
-    std::vector<V3D> indexed_qs;
-    double fit_error;
-    miller_ind.reserve(q_vectors.size());
-    indexed_qs.reserve(q_vectors.size());
-    IndexingUtils::GetIndexedPeaks(UB, q_vectors, tolerance, miller_ind,
-                                   indexed_qs, fit_error);
-
-    IndexingUtils::Optimize_UB(UB, miller_ind, indexed_qs, sigabc);
-
```

Comment:
@raquelalvarezbanos Yes. I believe that this was effectively called twice, once in `Find_UB` towards the end of the method and once in `FindUBUsingLatticeParameters`. I don't see why It needed to be called twice other than to get the errors for the lattice parameters, which is unnecessary now we pass the lattice as an argument. Removing it did not break any unit tests, which gave me some confidence that it is redundant. Perhaps someone with more knowledge of this code ( @AndreiSavici ?) could verify that second call is unnecessary.